app/routes/application.js

```diff
@@ -12,6 +13,137 @@ export default Ember.Route.extend(ApplicationRouteMixin, {
   isOnboarding: Ember.computed.alias('onboarding.isOnboarding'),
   onboardingRoute: Ember.computed.alias('onboarding.currentRoute'),
 
+  headTags: [
+    {
+      type: 'link',
+      tagId: 'link-canonical',
+      attrs: {
+        rel: 'canonical',
+        content: ENV.WEB_BASE_URL
+      },
+    },
+    {
+      type: 'meta',
+      tagId: 'meta-description',
+      attrs: {
+        property: 'description',
+        content: 'Contribute to software projects for social good. Give your time or money to help build software to better the arts, education, government, science, and more.'
+      },
+    },
+    {
+      type: 'meta',
+      tagId: 'meta-og-description',
+      attrs: {
+        property: 'og:description',
+        content: 'Contribute to software projects for social good. Give your time or money to help build software to better the arts, education, government, science, and more.'
+      },
+    },
+    {
+      type: 'meta',
+      tagId: 'meta-og-image',
+      attrs: {
+        property: 'og:image',
+        content: 'https://d3pgew4wbk2vb1.cloudfront.net/images/universal-card.png'
+      },
+    },
+    {
+      type: 'meta',
+      tagId: 'meta-og-site-name',
+      attrs: {
+        property: 'og:site_name',
+        content: 'Code Corps'
+      },
+    },
+    {
+      type: 'meta',
+      tagId: 'meta-og-title',
+      attrs: {
+        property: 'og:title',
+        content: 'Code Corps | Build a better future.'
```

Comment:
Not sure if this should stay if we are removing the other title, but it's not blocking 💄